packages/eset_protect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.2"
```

```suggestion
- version: "1.2.0"
```
We use semver, so enhancements bump minor, not patch.